docs/design/autoware-interfaces/ad-api/types/autoware_adapi_v1_msgs/msg/VehicleStatus.md

```diff
@@ -16,7 +16,6 @@ autoware_adapi_v1_msgs/Gear gear
 autoware_adapi_v1_msgs/TurnIndicators turn_indicators
 autoware_adapi_v1_msgs/HazardLights hazard_lights
 float64 steering_tire_angle
-float32 energy_percentage  # Battery percentage or fuel percentage, it will depends on the vehicle.
```

Comment:
Why is this being removed?

Reply:
Because its internal interface has not been defined. So I'll release it with a minimal set to avoid data changes later.
https://github.com/autowarefoundation/autoware.universe/pull/7464

Reply:
That means this is a left over changes that should have been fixed with the PR7464, correct?
In that case, we might want to split this into a separate PR. It seems like we are deleting this float at the same time with release PR.

Reply:
Ok so it was never released in the first place, if yes, I'm alright with this :+1: @mitsudome-r I'm marking this as resolved, as long as it is explained here, I'm ok with it being in this PR.